instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/patches/client.rb

```diff
@@ -15,6 +15,45 @@ module Patches
         module Client
           def query(sql, options = {})
             attributes = client_attributes
+            attributes.merge!(span_attributes(sql))
+
+            tracer.in_span(
+              span_name(sql),
+              attributes: attributes.merge!(OpenTelemetry::Instrumentation::Mysql2.attributes),
+              kind: :client
+            ) do
+              super(sql, options)
+            end
+          end
+
+          def prepare(sql)
+            attributes = client_attributes
+            attributes.merge!(span_attributes(sql))
+
+            tracer.in_span(
+              span_name(sql),
+              attributes: attributes.merge!(OpenTelemetry::Instrumentation::Mysql2.attributes),
+              kind: :client
+            ) do
+              super(sql)
+            end
+          end
+
+          private
+
+          def span_name(sql)
```

Comment:
```suggestion
          def _otel_span_name(sql)
```
Something that we haven't done yet and probably should is adopt a naming convention for private methods we add via monkey patches to a class.

We should start prefixing all private methods with `_otel` going forward.